.github/workflows/maintenance.yaml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron: '*/5 * * * *'
+  workflow_dispatch: # Enables on-demand/manual triggering
+jobs:
+  job:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - run: |
+          go get github.com/aws/aws-sdk-go
+          go mod tidy
+          go generate ./...
+          cd tools/provider-schema
+          terraform init -upgrade
```

Comment:
@wata727 @bendrucker anyone have any ideas why `terraform init -upgrade` doesn't seem to be upgrading the AWS provider here? https://github.com/patmyron/tflint-ruleset-aws/pull/2

That command upgrades the AWS provider successfully for me in other environments I've tried it...